docs/toc.md

```diff
@@ -55,6 +55,11 @@
 <!-- .NET Core Content -->
 # [.NET Core Guide](core/index.md)
 ## [Get started](core/get-started.md)
+### [Get started with C# and Visual Studio Code](core/tutorials/with-visual-studio-code.md)
```

Comment:
I was actually thinking about these topics this week and I think they belong under Get Started as you did here. Let's test this new location and if we feel that's the right spot, we should also move physical locations (again).